### PR TITLE
Rework to run in Fedora, not CentOS 

### PR DESCRIPTION
**Pull request by cgwalters**

### CODE REVIEW

**Comment by cgwalters** [comment]
@giuseppe Care to sanity check this?  Doesn't need a detailed review IMO

File: rdgo/rpkg-prep-sources [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python2
```
Comment (cgwalters) [inline rdgo/rpkg-prep-sources]: Note rpkg is still python2 even in Fedora.

File: rdgo/git.py [hunk 2]
```diff
@@ -125,16 +128,16 @@ def _list_submodules_in(self, checkout, uri, rev='HEAD'):
...
-            if line == '':
+            if line == b'':
                 continue
             line = line[1:]
-            parts = line.split(' ', 2)
+            parts = line.split(b' ', 2)
             if len(parts) < 2:
                 continue
-            sub_checksum, sub_name = parts[0:2]
+            sub_checksum, sub_name = map(lambda x: x.decode('UTF-8'), parts[0:2])
```
Comment (giuseppe) [inline rdgo/git.py]: would it make sense to decode `line` and leave this as before?  It is slightly more readable, not sure if it is worth another version

Comment (cgwalters) [reply rdgo/git.py]: Good suggestion, done thanks!

**Pull request merged**